notify.js

```diff
@@ -99,6 +108,11 @@
         try {
             new N('');
         } catch (e) {
+            if (SW) {
```

Comment:
Also is this possibly a false positive? Supporting Service Worker does not necessarily mean supporting push notifications.